tests/test_smiles_helpers.py

```diff
@@ -178,19 +179,19 @@
          (2, {'element': 'C', 'hcount': 1, 'charge': 0}),
          (3, {'element': 'C', 'hcount': 0, 'charge': 0}),
          (4, {'element': 'C', 'hcount': 3, 'charge': 0})],
-        [(0, 1, {'order': 1}),
+        [(0, 1, {'order': 2}),
          (1, 2, {'order': 1}),
-         (2, 3, {'order': 1}),
+         (2, 3, {'order': 2}),
          (3, 0, {'order': 1}),
          (3, 4, {'order': 1})],
         [(0, {'element': 'C', 'hcount': 1, 'charge': 0, 'aromatic': True}),
          (1, {'element': 'C', 'hcount': 1, 'charge': 0, 'aromatic': True}),
          (2, {'element': 'C', 'hcount': 1, 'charge': 0, 'aromatic': True}),
          (3, {'element': 'C', 'hcount': 0, 'charge': 0, 'aromatic': True}),
          (4, {'element': 'C', 'hcount': 3, 'charge': 0, 'aromatic': False})],
-        [(0, 1, {'order': 1}),
+        [(0, 1, {'order': 2}),
          (1, 2, {'order': 1}),
-         (2, 3, {'order': 1}),
+         (2, 3, {'order': 2}),
```

Comment:
Something is strange about this test case; Shouldn't this result into aromatic bond orders since it displays DIME. I guess the issue is that in contrast to the previous case now nodes have to be initially marked as aromatic.